examples/utils/trade.py

```diff
@@ -166,21 +174,21 @@ def getConfigs():
     pool_id = 2 if chain_id == 1729 else 4
     rpc_url = 'https://rpc.reya.network' if chain_id == 1729 else 'https://rpc.reya-cronos.gelato.digital'
     core_proxy_address = "0xA763B6a5E09378434406C003daE6487FbbDc1a80" if chain_id == 1729 else "0xC6fB022962e1426F4e0ec9D2F8861c57926E9f72"
-    multicall2_address = "0x82a55eB2346277d85D5D0A60101ebD0F02680B0D" if chain_id == 1729 else "0xd9f0F399f5264fAac91476Dbd950574726D18633"
+    multicall_address = "0xED28d27dFcA47AD2513C9f2e2d3C098C2eA5A47F" if chain_id == 1729 else "0x5abde4F0aF8Eaf3c9967f7fA126E59A103357b5C"
```

Comment:
why did we change this?

Reply:
we changed this to use our custom multicall that returns the actual error rather than the usual multicall2 failed message